src/shared/components/status-box/StatusBox.tsx

```diff
@@ -54,23 +54,23 @@ LoadError.displayName = 'LoadError';
 
 export const Loading: React.FC<LoadingProps> = ({ className }) => (
   <div
-    className={classNames('co-m-loader co-an-fade-in-out', className)}
+    className={classNames('hacDev-m-loader co-an-fade-in-out', className)}
```

Comment:
Also the `co-an-fade-in-out` class doesn't seem to be defined anywhere.

Reply:
removed it